src/Serialization/ConstructingMessageSerializerTest.php

```diff
@@ -24,11 +24,11 @@ public function serializing_messages_with_aggregate_root_ids()
         $inflector = new DotSeparatedSnakeCaseInflector();
         $aggregateRootIdType = $inflector->instanceToType($aggregateRootId);
         $timeOfRecording = (new TestClock())->pointInTime();
-        $message = new Message(new EventStub('original value'), [
+        $message = new Message(new PayloadStub('original value'), [
             Header::AGGREGATE_ROOT_ID      => $aggregateRootId,
             Header::AGGREGATE_ROOT_ID_TYPE => $aggregateRootIdType,
             Header::TIME_OF_RECORDING      => $timeOfRecording->toString(),
-            Header::EVENT_TYPE             => $inflector->classNameToType(EventStub::class),
+            Header::EVENT_TYPE             => $inflector->classNameToType(PayloadStub::class),
```

Comment:
I'd personally not use indentation for `=>` assignments because updates to these could make code reviews harder to parse. But feel free to ignore.